css/structure/jquery.mobile.forms.textinput.css

```diff
@@ -114,11 +114,14 @@ for Android 4.1 */
 .ui-input-text input[type=number]::-webkit-outer-spin-button {
 	margin: 0;
 }
-/* Resolves issue #5756: Textinput in IE10 has a default clear button */
-.ui-input-text input::-ms-clear,
-.ui-input-search input::-ms-clear {
+/* Resolves issue #8001: Default clear button in date input type*/
+.ui-hide-default-clear::-ms-clear,
+ {
 	display: none;
 }
+.ui-hide-default-clear::-webkit-clear-button{
+display: none;
```

Comment:
The `display: none` should be indented. 
http://contribute.jquery.org/style-guide/css/#spacing